scripts/02_prep_reporting/0160-load-bcfishpass-data.R

```diff
@@ -2,25 +2,16 @@
 ## simons db will not have the updated pscis names so we we include workflows to see that our new pscis crossings match our old modelled crossings
 
 source('R/packages.R')
-source('R/functions.R')
-source('R/private_info.R')
-
-# conn <- DBI::dbConnect(
-#   RPostgres::Postgres(),
-#   dbname = dbname_wsl,
-#   host = host_wsl,
-#   port = port_wsl,
-#   user = user_wsl,
-#   password = password_wsl
-# )
+#source('R/private_info.R')
+
 
 conn <- DBI::dbConnect(
   RPostgres::Postgres(),
-  dbname = dbname,
-  host = host,
-  port = port,
-  user = user,
-  password = password
+  dbname = Sys.getenv('PG_DB_BCBARRIERS'),
```

Comment:
our latest database is `PG_DB_DO` but this actually works because this is run with a `param_watersheds.csv` file ( https://github.com/smnorris/bcfishpass/blob/main/parameters/cwf/param_watersheds.csv ) that specifies `mad` (mean annual discharge` as the criteria used to screen hab suitability. This is what CWF is using so whatever.  I prefer channel width because I understand it better, helped derive the equation and because there are watershed groups (ex. Bulkley) where the MAD data used in not open sourced (proprietary from Foundary Geospatial) but CWF basically stole the FWCP fish passage project from me a few years ago likely using my shared proposal to get the contract , they use MAD for it and Elk is open source [Pacific Climate Impacts Consortium](https://www.pacificclimate.org/) https://github.com/smnorris/bcfishpass/blob/main/model/discharge/Makefile